packages/portfolio/src/adapters/dullahan-api/api-maker.ts

```diff
@@ -305,7 +306,8 @@ export const apiConfig: ApiConfig = freeze(
       tokenInfo: 'https://zero.yoroiwallet.com/tokens/info',
       tokenInfos: 'https://zero.yoroiwallet.com/tokens/info/multi',
       tokenTraits: 'https://zero.yoroiwallet.com/tokens/nft/traits',
-      tokenActivityUpdates: 'https://zero.yoroiwallet.com/defi/token-activity',
```

Comment:
created this ticket to update the endpoints later: https://emurgo.atlassian.net/browse/YOMO-1858

Reply:
done